subsys/bluetooth/services/alexa_gadgets/CMakeLists.txt

```
@@ -16,15 +16,21 @@
 # (https://jpa.kapsi.fi/nanopb/download/)
 
 
-# Check system for protoc and nanopb binaries presence.
+# Check system for protoc and nanopb binaries presence, needed to build the gadget profile files.
 # Prefer local variable to make version matching easier
-find_program(PROTOC_BIN protoc HINTS "${PROJECT_SOURCE_DIR}/*/generator-bin/")
-find_program(NANOPB_BIN protoc-gen-nanopb HINTS "${PROJECT_SOURCE_DIR}/*/generator-bin/")
+find_program(PROTOC_BIN
+    protoc
+    HINTS "${CMAKE_SOURCE_DIR}/*/generator-bin/" "${ZEPHYR_BASE}/../*/generator-bin/" "${ZEPHYR_BASE}/../tools/*/generator-bin/"
```

I suggest to line wrap those, to make it easier to see what are the hints:
```suggestion
    HINTS
    "${CMAKE_SOURCE_DIR}/*/generator-bin/"
    "${ZEPHYR_BASE}/../*/generator-bin/"
    "${ZEPHYR_BASE}/../tools/*/generator-bin/"
```